docs/guidelines/platforms/android/guideline_understandable_android.md

```diff
@@ -70,28 +70,69 @@ Note: Certain elements may require context-specific labels to convey their funct
 
 ---
 
-## Input Assistance
+## Input Assistance (WCAG 3.3)
 
 Help users avoid and correct mistakes.
 
-### Error identification
+### Error identification (WCAG 3.3.1 - Level A)
 
-*This technique covers point 3.3.1 Error Identification - Level A of the WCAG standard.*
+If an input error is automatically detected, the item that is in error is identified and the error is described to the user in text.
 
-:white_check_mark: **Success criteria**
+> This technique covers point *3.3.1 Error Identification - Level A of the WCAG standard.*
+
+#### ✅ Success technique(s)
 
 If an error occurs on the item the user is interacting with, the error should be clearly defined and described so the user can get clear information on why the error occurred and what to do in order to fix it.
 
 It is recommended to always use or extend system-provided widgets that are as far down Android's class hierarchy as possible because they already have the most accessibility capabilities that your app needs.
 
 On the other hand, if your app requires the implementation of custom components, you will need to implement [custom accessibility events](https://developer.android.com/guide/topics/ui/accessibility/principles#define-custom-events) to provide all accessibility updates that the system-provided widgets do.
 
-:no_entry_sign: **Failure criteria**
+For Views, `TextView` can be used to display error message. `TextInputLayout` can also be used, as it has built-in support for error messages.
+
+Example - showing error on `TextInputLayout`:
+
+```
+<!-- Error message for en-US locale would be "Invalid date, must be in the form DD/MM/YYYY, for example, 01/01/1990" -->
+ textInputLayout.error = getString(R.string.date_error)
+ textInputLayout.setErrorEnabled(true)
+```
+
+For Compose, a `error` semantics property can be used. However, this will only announce the error message, but it won't be shown on the screen.
```

Comment:
```suggestion
For Compose, an `error` semantics property can be used. However, this will only announce the error message, but it won't be shown on the screen.
```